coordinator/src/main/java/org/apache/uniffle/coordinator/ServerNode.java

```diff
@@ -44,6 +51,21 @@ public ServerNode(
       int eventNumInFlush,
       Set<String> tags,
       boolean isHealthy) {
+    this(id, ip, port, usedMemory, preAllocatedMemory, availableMemory, eventNumInFlush, tags, isHealthy,
+        Maps.newHashMap());
+  }
+
+  public ServerNode(
+      String id,
+      String ip,
+      int port,
+      long usedMemory,
+      long preAllocatedMemory,
+      long availableMemory,
+      int eventNumInFlush,
+      Set<String> tags,
+      boolean isHealthy,
+      Map<String, RssProtos.StorageInfo> storageInfoMap) {
     this.id = id;
```

Comment:
Is this feature a compatible feature? Can coordinator  recognize the old server's request?

Reply:
> Yes. the `storageInfoMap` should be an empty Map when deserializing from old code.

Could the default value be null?

Reply:
OK.